packages/core/src/Tab/constants.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
```

Comment:
@James300 @craigpalermo we need to work out a convention for where this sort of thing lives and can be ignored from coverage - fake content to be shared between stories and tests is the category

Reply:
^Still an issue here yep😆

Reply:
maybe DocsUtils if that stays in core